send_messages.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+import can
+
+def transmit_can_message(channel="can0", bustype="socketcan"):
+    """
+    Initializes the CAN bus, prompts the user for message parameters,
+    and transmits the CAN message.
+    """
+    # Initialize the CAN bus interface
+    bus = can.interface.Bus(channel=channel, bustype=bustype)
+
+    # Prompt for the arbitration ID (CAN ID)
+    arb_id_input = input("Enter arbitration id (e.g. 0x123): ").strip()
```

Comment:
```suggestion
    arb_id_input = input("Enter 3-digit hex message id (e.g. 0x123): ").strip()
```
nit. If we're expecting 3-digit hex, let's clarify